index.js

```diff
@@ -100,32 +131,39 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
-
-// console.log(myMap([1,2,3,4,5], triple)) 
-// // => [3,6,9,12,15]
+console.log("%cCallbacks - Question 1", "color: red")
+function triple(element) {
+  return element * 3
+}
+console.log(myMap([1,2,3,4,5], triple)) 
+// => [3,6,9,12,15]
 
-// console.log(myMap([2,4,6,8], triple))   
-// // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
+console.log(myMap([2,4,6,8], triple))   
+// => [6,12,18,24]
+console.log("%c----------", "color: red") 
 
 
 // ***** Callbacks - Question 2 *****
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 2", "color: red")
-
-// console.log(myMap(["Raffy", "Chase"], greet)) 
-// // => ["Hi, Raffy!", "Hi, Chase!"]
-// console.log("%c----------", "color: red") 
+console.log("%cCallbacks - Question 2", "color: red")
+function greet(element) {
+  return `Hi, ${element}!`
+}
+console.log(myMap(["Raffy", "Chase"], greet)) 
+// => ["Hi, Raffy!", "Hi, Chase!"]
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Scope & Closures *****
 
-function takeANumber(line, name) {
-  line.push(name)
+let num = 1
+
+function takeATicketNumber(line) {
 
-  return `Welcome, ${name}. You are number ${line.length} in line.`
+  line.push(num)
+  num++
+  return `Welcome. You are ticket number ${line.length}.`
```

Comment:
If you're customer number 100, and the line is empty, they'll receive that their ticket number is 1.
So, you should be returning the `num` in the interpolation rather than the `line.length`.
But doing that is going to affect the order of your code, so think about how you'd change that!